snuba/utils/streams/batching.py

```diff
@@ -15,13 +15,13 @@
     TypeVar,
 )
 
-from snuba.utils.metrics import MetricsBackend
-from snuba.utils.streams.processing.strategies.abstract import (
+from streaming_kafka_consumer.processing.strategies.abstract import (
```

Comment:
Doesn't this file move ?

Reply:
I was on the fence about this one. I can move it over too.